cmd/mps-control-daemon/mps/daemon.go

```diff
@@ -216,18 +216,18 @@ func (m *Daemon) perDevicePinnedDeviceMemoryLimits() map[string]string {
 	totalMemoryInBytesPerDevice := make(map[string]uint64)
 	replicasPerDevice := make(map[string]uint64)
 	for _, device := range m.Devices() {
-		uuid := device.GetUUID()
-		totalMemoryInBytesPerDevice[uuid] = device.TotalMemory
-		replicasPerDevice[uuid] += 1
+		index := device.Index
```

Comment:
I believe this is related to the discussion above, but what will the indices look like when we are iterating over MIG devices?